static/styles/compose.css

```diff
@@ -229,8 +239,6 @@
     left: 0;
     z-index: 4;
     width: 100%;
-
-    background-color: hsl(0, 0%, 100%);
```

Comment:
Any selector that has its old background color removed is now inheriting the new background color.